### DISCUSSION TITLE
Poll : Does the software work as expected ?

### DISCUSSION THREAD

**Discussion by Jyben** [Jyben/cs2-mm-server-picker#99] [Polls]
I haven't played the game for a long time. 

With recent bugs and recent feedback I wonder if the software is still actually blocking relay servers. Normally it doesn't give a 100% chance of hitting the desired server, but it should at least increase them. 

Do you see this improvement using the software? Depending on your feedback, I will see if I delete/archive the software

**Comment by rekt0ro** [comment]
Hello!
Can you please update your software it's not working anymore! Thank you.
